### PR TITLE
doc: Remove CCT from README and release process docs

### PR DESCRIPTION
**Pull request by barton2526**
it's dead :(

### CODE REVIEW

**Comment by barton2526** [comment]
Should we add Discord to the README so it doesn't feel empty?

**Comment by Pythonix** [comment]
> Should we add Discord to the README so it doesn't feel empty?

IMHO we should add Discord.

**Review by jamescowens — CHANGES_REQUESTED** [review]
Please add Discord.

**Comment by div72** [comment]
Freenode link could also be updated to liberachat.

**Comment by barton2526** [comment]
> 
> 
> Freenode link could also be updated to liberachat.

The reason I didn't go through with this is because we are currently operating both irc networks.  I'm not sure which has more users.

File: doc/README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ To download Gridcoin, visit [gridcoin.us](https://gridcoin.us).
 * See the documentation at the [Gridcoin Wiki](https://wiki.gridcoin.us/Main_Page)
 for help and more information.
 * A lot of features core features are based on Bitcoin and have been documented on the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Main_Page)
-* Ask for help or discuss on [#gridcoin](https://webchat.freenode.net?channels=gridcoin) on Freenode.
-* Ask for help or discuss on the [Cryptocurrencytalk](https://cryptocurrencytalk.com/forum/464-gridcoin-grc/) forums
+* For general questions, please visit our Discord server at https://discord.gg/jf9XX4a
+* Ask for help or discuss on [#gridcoin](https://webchat.freenode.net?channels=gridcoin) on Freenode
 * You can also join us on [Slack](https://grcinvite.herokuapp.com/) with the invite Token GRCsquad
```
Comment (RoboticMind) [inline doc/README.md]: Should probably switch the slack link to use the link used everywhere else
```suggestion
* You can also join us on [Slack](https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTE4N2I3ZWZjYWJlZGM1Zjg3MTUyMDhiN2M5NmRmZTA2NDA0ZmY1ZTFmOGM3ZGU2YTBkOTdhNTk2ZjkzMGZkODY/)```

**Comment by jamescowens** [comment]
Maybe we should put both?

**Review by jamescowens — APPROVED** [review]
ACK

**Comment by jamescowens** [comment]
We are going to leave as is.

**Pull request merged**